### ISSUE TITLE
Change connectivity package to connectivity_plus to support Windows 

### ISSUE THREAD

**Issue by gcostaapps** [parse-community/Parse-SDK-Flutter#619]
Hi, 

I've tested app that I'm developing for web on Windows and everything worked fine except for the live queries, I checked that the problem is in the connectivity package that doesn't support Windows. I switched to the connectivity_plus 1.0.1 (supports null safety) and tested in a local branch and everything works fine.

It would be great if this feature were add here, it's just a matter of changing the pubspec.yaml and the import, no additional code is needed.

(Currently I'm developing an web app but the development experience for web is still very bad because of the lack of hot reload. So I choose to develop and do changes in the design on Windows and after finishing it publish it for web.)

**Comment by narratorek** [comment]
Hi gcostaapps,

> I switched to the connectivity_plus 1.0.1

How exactly did you do it? I have added connectivity_plus 1.0.1 to pubspec.yaml and it still doesn't work.
I still have communicates: 

```
[ERROR:flutter/lib/ui/ui_dart_state.cc(186)] Unhandled Exception: MissingPluginException(No implementation found for method check on channel plugins.flutter.io/connectivity)
#0      MethodChannel._invokeMethod
package:flutter/…/services/platform_channel.dart:156
<asynchronous suspension>
#1      MethodChannelConnectivity.checkConnectivity
package:connectivity_platform_interface/src/method_channel_connectivity.dart:41
<asynchronous suspension>
#2      Parse.checkConnectivity
package:parse_server_sdk_flutter/parse_server_sdk.dart:106
<asynchronous suspension>

════════ Exception caught by services library ══════════════════════════════════
The following MissingPluginException was thrown while activating platform stream on channel plugins.flutter.io/connectivity_status:
MissingPluginException(No implementation found for method listen on channel plugins.flutter.io/connectivity_status)

When the exception was thrown, this was the stack
#0      MethodChannel._invokeMethod
package:flutter/…/services/platform_channel.dart:156
<asynchronous suspension>
#1      EventChannel.receiveBroadcastStream.<anonymous closure>
package:flutter/…/services/platform_channel.dart:545
<asynchronous suspension>
```

**Comment by g-scosta** [comment]
You need to delete the old connectivity package from pubspec and replace in the code the import to point to the new one. Did you do that?

**Comment by narratorek** [comment]
I didn't have any connectivity package added in pubspec.yaml. However I added connectivity_plus: ^1.0.1 and imported it in every file, where I use parse_server_sdk_flutter 3.0.0. Nothing has changed.

**Comment by gcostaapps** [comment]
Hi @narratorek , if you didn't find any connectivity package I think that you added the pubspec.yaml to the dart package of the library. The dart package doesn't need to be modified, only the flutter one. If you go inside the flutter folder in packages you will find that the pubspec.yaml from there has this line

`
connectivity: ^2.0.2 # only used in the flutter part
`

Just replace that for the latest version of connectivity_plus and change the import in the file packages/flutter/lib/parse_server_sdk.dart .

[Here](https://github.com/gcostaapps/Parse-SDK-Flutter/commit/8f06fe0517cc3b79223763c8c4074e9f88c1b594) is a commit that I added this support in the fork that I did. There you will find the affected files.
